### DISCUSSION TITLE
Renovate doesn't create pull requests for (dev)dependencies

### DISCUSSION THREAD

**Discussion by MartinX3** [renovatebot/renovate#18812] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Only for 2 dependencies it creates a PR.
But not for the others.

As example it doesn't create a PR to upgrade `@yeger/vue2-masonry-wall` from `^2.0.65` to `^2.0.68` or `@babel/core` from `^7.19.6` to `^7.20.2`

It happens here
https://github.com/Rad-Touristik-Club-Koln-e-V-1972/Rad-Touristik-Club-Koln-e-V-1972.github.io

### Relevant debug logs

<details><summary>Logs</summary>

```
        {
            "depType": "dependencies",
            "depName": "@yeger/vue2-masonry-wall",
            "currentValue": "^2.0.65",
            "datasource": "npm",
            "prettyDepType": "dependency",
            "depIndex": 1,
            "updates": [],
            "warnings": [],
            "versioning": "npm",
            "sourceUrl": "https://github.com/DerYeger/vue2-masonry-wall",
            "homepage": "https://vue2-masonry-wall.yeger.eu",
            "currentVersion": "2.0.68"
          },
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by viceice** [comment, marked as answer]
expected, checkout range strategy inside our docs.

**Reply by MartinX3** [reply]
I wonder why the default value of `rangeStrategy` is not `auto`.
And the tutorial didn't navigate me to this option.
I would need to know this option to find it in the docs.

**Reply by viceice** [reply]
the default is auto, but we changed that a little since v33

**Reply by MartinX3** [reply]
Are the docs outdated?
https://docs.renovatebot.com/configuration-options/#rangestrategy

So `auto` ignores the npm strategy since v33?
I use caret `^` versioning and it ignores minor and patch updates.

**Reply by MartinX3** [reply]
Ah, after reading the docs again in combination with the v33.0.0 release notes I think I should set the rangeStrategy to `bump`
Thank you

**Reply by viceice** [reply]
yes, otherwise renovate will only widen if new version falls outside of the range by default.

**Reply by MartinX3** [reply]
@viceice do you know why it automerged a major version jump?
I copied the auto merge rules to avoid major version jumps from the docs.

https://github.com/Rad-Touristik-Club-Koln-e-V-1972/Rad-Touristik-Club-Koln-e-V-1972.github.io/pull/327

the PR also had a failed github but the entire check section somehow disappeared from the merged PR.
I wonder if the renovate bot ignores failed checks in PRs.

**Reply by viceice** [reply]
because you had global automerge enabled

https://github.com/Rad-Touristik-Club-Koln-e-V-1972/Rad-Touristik-Club-Koln-e-V-1972.github.io/commit/3b4244a058c574b4e7e25d364a955f6bb8d55fb7

**Reply by MartinX3** [reply]
Ah, it was a bit confusing for me, because https://docs.renovatebot.com/renovate-schema.json doesn't include the `"automerge"` key in the `"packageRules"` field.
So I didn't know if either the docs or the schema were correct.

But shouldn't renovate avoid a merge if the running checks were not okay / red?

**Reply by viceice** [reply]
yes, schema is autogenerated and far from perfect. so docs are right.

**Reply by MartinX3** [reply]
Ah, thank you.

And is there a ways to prevent automerge from merging if the github checks in a PR are running/failing?

**Reply by viceice** [reply]
make them mandatory on github, otherwise they are probably missing when renovate checks for automerge

read this https://docs.renovatebot.com/key-concepts/automerge/

**Reply by MartinX3** [reply]
Argh, I searched for `automerge` and `check` instead of `tests`.
Thank you!
